Prog/Employment Trends.sas

```diff
@@ -167,6 +179,19 @@ data industryjobs;
 	%mend yearloop;
 	%yearloop;
 
+	if naics = "1012" then icode = 1;
```

Comment:
This isn't quite right because the 4-digit NAICS codes provided do not add up to the 3-digit subtotals in the file. In other words, there are industries omitted from the 4-digit codes. So, you have to calculate the residual for the two "other" categories.

Reply:
@ptatian I see. I'll work on getting that in there.